### PR TITLE
Use File Scheme in FileSystemMasterTest

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Review by madanadit — APPROVED** [review]

File: core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 1]
```diff
@@ -2511,7 +2511,7 @@ public void unmountNonExistingPathWithException() throws Exception {
    */
   private AlluxioURI createTempUfsDir(String ufsPath) throws IOException {
     String path = mTestFolder.newFolder(ufsPath.split("/")).getPath();
-    return new AlluxioURI(path);
+    return new AlluxioURI("file:??", null, path);
```
Comment (madanadit) [inline core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java:2514]: ??

Comment (madanadit) [reply core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java:2514]: this should be just `file` for the scheme?

**Review by madanadit — CHANGES_REQUESTED** [review]
Is the scheme specified correctly?

File: core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java (hunk 1 above)
Comment (yuzhu) [reply core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: 🤦‍♂️ 🤦‍♂️  sticky shift keys....

File: core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 2]
```diff
@@ -2511,7 +2511,7 @@ public void unmountNonExistingPathWithException() throws Exception {
    */
   private AlluxioURI createTempUfsDir(String ufsPath) throws IOException {
     String path = mTestFolder.newFolder(ufsPath.split("/")).getPath();
-    return new AlluxioURI("file:??", null, path);
+    return new AlluxioURI("file://", null, path);
```
Comment (madanadit) [inline core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java:2514]: I think just `file` and not `file://` is sufficient. That's how it used in `AlluxioUriTest`

**Review by madanadit — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**